opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/ipfix.adoc

```diff
@@ -24,8 +24,11 @@ The IPFIX UDP Parser supports protocol detection.
 .Parameters for the IPFIX UDP Parser
 [options="header, autowidth"]
 |===
-| Parameter         | Description                                                                | Required | Default value
-| `templateTimeout` | Templates must be re-declared in the given duration or the will be dropped | required | 30 minutes
+| Parameter             | Description                                                                    | Required | Default value
+| `templateTimeout`     | Templates must be re-declared in the given duration or they will be dropped    | no       | 30 minutes
+| `maxClockSkew`        | The maximum delta in seconds between exporter and Minion timestamps.       | no       | 0
+| `clockSkewEventRate`  | Used to rate-limit clock skew events in seconds.                           | no       | 3600
+| `lookupsEnabled`      | Used to enable or disable DNS resolution for flows.                        | no       | true
```

Comment:
Now that we're referring to this property on the adapter, should we change it to `dnsLookupsEnabled` instead? (Just to be clear in case we add other types of lookups / enrichment later.)

Reply:
Works for me. Done